docs/conf.py

```diff
@@ -168,7 +170,12 @@ def linkcode_resolve(domain, info):
     else:
         linespec = ""
 
-    fn = relpath(fn, start=dirname(SimPEG.__file__))
+    # Exception for building locally on Windows when Python on C: drive and repo on D: drive
+    # Note that these links will not work on the local build!!!
+    try:
+    	fn = relpath(fn, start=dirname(SimPEG.__file__))
+    except Exception:
```

Comment:
Don't catch general `Exception` here, I think you're wanting to specifically catch and ignore the error due to a relative path no being possible on windows when the start and end are on two different drives. (A silly limitation of Windows). But the exception that is raised in those cases is a `ValueError`, so specifically except a `ValueError`

Reply:
sure